docs/installation.md

```diff
@@ -31,6 +31,11 @@ overidden by setting the `HX_DEPLOY_CONFIG` environment variable.
 
 The configuration file must be json formatted, and be consistent with the
 [ToolConfig][toolconfig-adl] ADL type (ADL is the [Algebraic Data Language][adl]).
+[An example configuration file can be found at: ](docs/example_deploy.json)
+
+Additionally, aws credentials must be supplied and will be expected at $HOME/.aws/credentials
```

Comment:
This is only partially correct. AWS credentials must indeed be provided, but this can be done by any of the standard AWS mechanisms:

   -  the $HOME/.aws/credentials file
   -  the standard environment variables
   -  EC2 instance metadata

it's the latter we always rely on for deployed systems.

Reply:
I'll add the other ones. 
The issue is that it isn't clear from the doco that it is expected, or how to use the other mechanisms, or if it is automatic.
Is it automatic as part of the aws lib?

Reply:
Yes. this code relies on `Credentials Discover` mode in the amazonka library:

http://hackage.haskell.org/package/amazonka-1.6.1/docs/Network-AWS-Auth.html